server/index.js

```diff
@@ -67,13 +67,14 @@ app.get('/health', (req, res) => {
 
 // function to catch get req from client login
 app.get('/user/login', (req, res) => {
-  console.log(req.body);
-  dbHelpers.getUserByGivenUsername(req.query.username, (err, user) => {
+  console.log(req.query);
+  dbHelpers.getUserByUsername(req.query.username, (err, user) => {
```

Comment:
to keep consistency we should keep all the methods names similar
change this back to getUserByGivenUsername and just change the name of it in the required file